storage/flux/table.go

```diff
@@ -196,6 +196,10 @@ func (t *table) readTags(tags models.Tags) {
 
 	for _, tag := range tags {
 		j := execute.ColIdx(string(tag.Key), t.cols)
+		// In the case of group aggregates, there might be tags that are excluded so you cannot find it.
+		if j < 0 {
+			continue
+		}
```

Comment:
Can you explain what this is doing a bit more?

Reply:
In the case of group aggregate, tags that are not referenced in `group()` are not included in the result, but `readTags ()` still get a complete tag list. Here is just to skip the tags that should not present in the result.